### DISCUSSION TITLE
Debug FL Studio "ilbridge.exe" issue in Sandboxie

### DISCUSSION THREAD

**Discussion by J-Wildfire-T** [sandboxie-plus/Sandboxie#1755] [Documents and notes]
**Describe the bug**
Is there a way I can find out, maybe through a debug dump or something similar, why when running FL Studio inside a sandbox it wont launch ilbridge.exe.  All I get is FLStudio telling me there was a problem.  I know the plugins work fine in FLStudio outside a sandboxed instance.

**To Reproduce**
Launch FL Studio and try to load a 32bit plugin in a 64bit instance.

**Expected behavior**
I'm not 100% certain on the exact on the process behind the scenes but it should load ilbridge.exe and import the VST plugin in a wrapper.

**Screenshots**
Screenshot wont show anything more than a message saying there was a problem opening the plugin.

**System details and installed software (please provide the following information):**
Windows 10 LTSC v1809 Build 17763.1971
Sandboxie Plus 0.7.5 x64.
It's been doing this since from when I first started using Sandboxie over 2-3 years ago.
ESET Internet Security v14.1.20.0

**Additional context**
N/A

**Sandboxie configuration**
N/A

**Comment by DavidXanatos** [comment]
You can use the resource log as well as the tracing options in the settings,
you can also try to go all in on the debugging front and disable parts of the sandbox isolation,

you can add:
```
[DebugOptions]
Option1=OriginalToken|y|Keep the original unrestricted tocken
Option2=OpenToken|y|Use a unrestricted and unfiltered token duplicate
Option3=UnrestrictedToken|y|-Don't restrict the sandboxed token
Option4=KeepTokenIntegrity|y|--Keep token integrity level
Option5=AnonymousLogon|n|--Don't set the anonymous SID
Option6=UnfilteredToken|y|-Don't filter the sandboxed token
Option7=NoSandboxieDesktop|y|Don't proxy desktop operations
Option8=NoSysCallHooks|y|Don't hook system calls
```

or https://github.com/sandboxie-plus/Sandboxie/issues/2134#issuecomment-1215466315

to your Sandboxie-Plus.ini to enable additional debug options in the Plus UI

you can try for example:
OriginalToken=y
NoSysCallHooks=y
NoSandboxieDesktop=y

that pretty much disables most of the driver level isolation

**Comment by J-Wildfire-T** [comment]
I'll be trying these in the next few days.

Interestingly enough I do not have this issue on a Windows 7 machine with no firewall installed but isolated to the local network.
I'm going to try a new W10 install with stock security settings.

**Update**
Tried it on a stock install of W10, same issue.
